docs/layers.md

```diff
@@ -879,16 +890,20 @@ To resolve inconsistency in data tagging in OpenStreetMap we normalize several o
 * `fuel` - Fuel stations provide liquid gas (or diesel) for automotive use.
 * `furniture`
 * `gallery` - An art gallery.
+* `gaming`
 * `garden` - _See planned bug fixes in [#1085](https://github.com/tilezen/vector-datasource/issues/1085)._
 * `gardener`
+* `garden_centre`
 * `gas_canister` - Shop selling bottled gas for cooking. Some offer gas canister refills.
 * `gate` - with `kind_detail` one of `chain`, `gate`, `kissing_gate`, `lift_gate`, `stile`, `swing_gate`.
 * `generator`
 * `geyser`
 * `gift`
+* `golf` - Shop selling golf equipment.
 * `golf_course`
 * `government`
-* `greengrocer`
+* `greengrocer` - Shop selling fruits and vegetables.
+* `grocery` - Shop selling non-perishable foods.
```

Comment:
`Shop selling non-perishable foods, also known as a "supermarket".`?

Reply:
I changed it to:

> Shop selling non-perishable food often similar to, but smaller than, a `supermarket`. See also [grocery store on Wikipedia](https://en.wikipedia.org/wiki/Grocery_store).

I get the impression from that Wikipedia article that a grocery store doesn't generally sell fresh fruits and vegetables, nor non-food items. I can't remember ever having been in a store which had such a laser focus - all the ones which come to mind either additionally sold stationery, cards, household products and similar non-food items (and might be called a newsagents or convenience store), or had a fridge/freezer section for selling perishable food.

Should we merge this into `supermarket` or `convenience`?

Reply:
In the USA `grocery` is probably more akin to "corner store" or "bodega" versus a supermarket which is a usually "big box" style store. Fine as-is from OSM.